freeipa-api/src/main/java/com/sequenceiq/freeipa/api/v1/freeipa/stack/model/scale/DownscaleResponse.java

```diff
@@ -10,8 +12,20 @@
 @JsonInclude(JsonInclude.Include.NON_NULL)
 public class DownscaleResponse extends ScaleResponseBase {
 
+    private Set<String> removedNodes;
```

Comment:
I tihnk `SelectedNodes` / `DownscaleCandidates` or similar would be better, as if the downscale fails these won't be removed, and also at this point these are not removed

Reply:
Yes, sounds way better, thank you.